query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_15581.rs

```diff
@@ -61,6 +61,7 @@ mod prisma_15581 {
         .to_owned()
     }
 
+    // TODO: This test works locally, but seem to fail on the CI. Investigate.
```

Comment:
TODO: remove this comment with a `[skip ci]` commit message once we get green CI

Reply:
(Doing this after at least one other person has seen the green CI)

Reply:
```suggestion
```